src/site/stages/build/process-cms-exports/schemas/transformed/taxonomy_term-administration.js

```diff
@@ -1,13 +1,56 @@
-const contentModelType = 'taxonomy_term-administration';
+/* eslint-disable camelcase */
+
+const nullableString = { type: ['string', 'null'] };
```

Comment:
We might want this in the `common` schemas directory, I think I've seen this in other schemas as well.

Reply:
Yeah, it was simple enough I didn't want to add it to `common` and load up another schema to reference from within AJV. :man_shrugging: I don't have a strong opinion. We can make that happen easily enough later if we wanted to.

Honestly, I was just being lazy because I had a bunch of nullable strings.